src/collar/oc_meshlabel.lsl

```diff
@@ -181,40 +191,64 @@ integer LabelsCount() {
             llSetLinkPrimitiveParamsFast(iLink,[PRIM_DESC,"Label~notexture~nocolor~nohide~noshiny"]);
         } else if (sLabel == "LabelBase") g_lLabelBaseElements += iLink;
     }
-    integer i;
-    for (i=0; i<llGetListLength(g_lLabelLinks);++i)
+    
+    if (bMultiLine && llGetListLength(lSingleParamLinks) > 0) // if we have multible lines, check if all prims are correctly named
     {
-        list lLineLinks = llParseString2List(llList2String(g_lLabelLinks,i),["|"],[]);
-        g_lCharLimit = llListReplaceList(g_lCharLimit,[llGetListLength(lLineLinks) * faces],i,i);
+        g_sErrorMsg += "Error! Some of your label prims don't have the line parameter in the name! (Should be: MeshLabel~num~line) \n";
+        integer i;
+        g_sErrorMsg += "Missing parameter in link numbers:";
+        for (i=0; i<llGetListLength(lSingleParamLinks);++i)
+        {
+            g_sErrorMsg += " "+llList2String(lSingleParamLinks,i);
```

Comment:
Not properly identifying the link numbers. In my test the actual output of this was different from the true link numbers. It indicated that 2 4 5, didn't have line numbers, while those links were actually: Links: 11 9 12